### DISCUSSION TITLE
add polat directori

### DISCUSSION THREAD

**Discussion by mostafamardani66** [Chia-Network/chia-blockchain#6940] [Support]
hi add polat directory dosent work

**Comment by loppefaaret** [comment]
you mean the menu does not open up, or does it simply not add the directory when you select it ? do you see any kind of error message or anything else when you try to add plot directory ?

**Comment by mostafamardani66** [comment]
On Wed, Jun 23, 2021 at 19:04 loppefaaret ***@***.***> wrote:

> you mean the menu does not open up, or does it simply not add the
> directory when you select it ? do you see any kind of error message or
> anything else when you try to add plot directory ?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/Chia-Network/chia-blockchain/discussions/6940#discussioncomment-910546>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AMLWHDWF42CFNMEABKOZKHDTUHWGJANCNFSM47F5BACA>
> .
>
not add the directory when i want add

**Reply by loppefaaret** [reply]
try and restart the program

**Comment by BrandtH22** [comment]
@mostafamardani66 , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
